tasks/77_expressions-matter/expressions-matter.js

```diff
@@ -0,0 +1,13 @@
+export function expressionMatter(...arr) {
+  let result = [];
+  arr = arr.sort((a, b) => a - b);
```

Comment:
there is a point in tasks that you don`t have to change order of parameters, when you so you will find out that there is some broken tests :)